<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ErrosionRectDialog</class>
 <widget class="QDialog" name="ErrosionRectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>756</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string/>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <property name="spacing">
         <number>2</number>
        </property>
        <item row="0" column="0">
         <widget class="LabelErosionDilatePoints" name="labelRectPoint">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(120, 120, 120);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxBtn">
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <property name="horizontalSpacing">
         <number>2</number>
        </property>
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>108</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButtonOther">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>41</height>
             </size>
            </property>
            <property name="text">
             <string>其他</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButtonCirclee">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>41</height>
             </size>
            </property>
            <property name="text">
             <string>圆</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButtonDiamond">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>41</height>
             </size>
            </property>
            <property name="text">
             <string>菱形</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnSelectAll">
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>51</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>75</width>
              <height>51</height>
             </size>
            </property>
            <property name="text">
             <string>全选</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnSelectAllNo">
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>51</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>75</width>
              <height>51</height>
             </size>
            </property>
            <property name="text">
             <string>全不选</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnQuit">
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>51</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>75</width>
              <height>51</height>
             </size>
            </property>
            <property name="text">
             <string>退出</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LabelErosionDilatePoints</class>
   <extends>QLabel</extends>
   <header>BinaryzationModule/Roi/LabelErosionDilatePoints.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
